app/commands/github/task/sync_task.rb

```diff
@@ -20,6 +21,16 @@ def claimed?
         labels.any? { |label| label.name == Github::IssueLabel.for_type(:status, :claimed) }
       end
 
+      def labeled?
+        types = %i[action knowledge module size type]
+
+        labels.any? do |label|
+          types.any? do |type|
+            label.of_type?(type)
+          end
+        end
```

Comment:
I could use `.product` here, but I don't know which one you prefer.

Reply:
Once we validate the label values we'll also need to check whether it actually has correct label values, and if not destroy it.

Reply:
In fact, I think we can **only** do this after the labels have been resynced, else we're analyzing off old data at this stage.